docs/en/04_Changelogs/4.0.0.md

```diff
@@ -956,6 +956,9 @@ A very small number of methods were chosen for deprecation, and will be removed
 * `DataObject::validateModelDefinitions()` has been removed. Validation and parsing of config is now handled
   within `DataObjectSchema`.
 * `CMSBatchAction_Delete` removed. Use `CMSBatchAction_Archive` instead.
+* `Query::first()` and `Query::current()` removed. Use `Query::record()` instead.
+* `Query::seek()` removed. To re-set the position, re-execute `Query::getIterator()` instead.
```

Comment:
Yeah, great eh? I think we only included the `seek()` behaviour to deal with the weirdness of using an `Iterator` rather than an `IteratorAggregate`.